packages/examples/packages/cronjob-duration/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { OnCronjobHandler } from '@metamask/snaps-sdk';
+import { panel, text, heading, MethodNotFoundError } from '@metamask/snaps-sdk';
+
+/**
+ * Handle cronjob execution requests from MetaMask. This handler handles one
+ * method:
+ *
+ * - `execute`: The JSON-RPC method that is called by MetaMask when the cronjob
+ * is triggered. This method is specified in the Snap manifest under the
+ * `endowment:cronjob` permission. If you want to support more methods (e.g.,
+ * with different times), you can add them to the manifest there.
+ *
+ * @param params - The request parameters.
+ * @param params.request - The JSON-RPC request object.
+ * @returns The JSON-RPC response.
+ * @see https://docs.metamask.io/snaps/reference/exports/#oncronjob
+ */
+export const onCronjob: OnCronjobHandler = async ({ request }) => {
+  switch (request.method) {
+    case 'execute':
+      // Cronjobs can execute any method that is available to the Snap.
+      return snap.request({
```

Comment:
Wondering if we should do something that is less obnoxious than the dialog. Maybe a notification?